client/src/pages/Checkout.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from "react";
+import { usePageLoaded } from "../hooks";
+import {
+  CardElement,
+  Elements,
+  Stripe,
+  useStripe,
+  useElements,
+} from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+import { createPaymentIntent } from "../services";
+
+const showCart = (cart) => {
+  // only show most recent
+  const item = cart[0];
```

Comment:
I guess we only have one type of cart item right now or I'd say just map through the items

Reply:
That's the idea, I wanted to create something that would be quick, but easy to extend.